### ISSUE TITLE
Please add readline capability for interactive mode

### ISSUE THREAD

**Issue by vimagick** [kardolus/chatgpt-cli#39]
The readline mode can be archived by [rlwrap][1], but it's much better to have it built-in.

```bash
$ rlwrap chatgpt -i
[2024-04-01 00:00:00] Q1: hello world┃ # press Ctrl-A to move the cursor to the beginning
[2024-04-01 00:00:00] Q1: ┃hello world 
```

Thanks!

[1]: https://github.com/hanslub42/rlwrap

**Comment by kardolus** [comment]
@vimagick thanks for the suggestion. I will look into this. Seems like a cool feature to have.

**Comment by kardolus** [comment]
@vimagick fixed this in version `v1.5.3`. Please let me know if you have any issues!
